hedera-node/hapi/build.gradle.kts

```diff
@@ -23,6 +23,39 @@ plugins {
 
 description = "Hedera API"
 
+val hapiProtoBranchOrTag = "v0.44.0"
+val hederaProtoDir = layout.projectDirectory.dir("hedera-protobufs")
+
+if (!gradle.startParameter.isOffline) {
+    @Suppress("UnstableApiUsage")
+    providers
+        .exec {
+            if (!hederaProtoDir.dir(".git").asFile.exists()) {
+                workingDir = layout.projectDirectory.asFile
+                commandLine(
+                    "git",
+                    "clone",
+                    "https://github.com/hashgraph/hedera-protobufs.git",
+                    "-q"
+                )
+            } else {
+                workingDir = hederaProtoDir.asFile
+                commandLine("git", "fetch", "-q")
+            }
+        }
+        .result
+        .get()
+}
+
+@Suppress("UnstableApiUsage")
+providers
+    .exec {
+        workingDir = hederaProtoDir.asFile
+        commandLine("git", "checkout", hapiProtoBranchOrTag, "-q")
+    }
+    .result
+    .get()
+
```

Comment:
This will most likely be replaced with some other solution for how to integrate `hedera-protobufs` soon. Doing this, instead of using `me.champeau.includegit`, makes the current approach fully configuration cache compatible (and we did not use any other features of that plugin).